common/bytes.go

```diff
@@ -14,7 +14,6 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package common contains various helper functions.
```

Comment:
This "unhelpful comment" is the Godoc synopsis of package common and should not be deleted.